### DISCUSSION TITLE
有无计划开发macOS版本？

### DISCUSSION THREAD

**Discussion by Mrered** [AlistGo/alist#1896] [General]
如题，有无计划开发基于macOS的版本。
想用自己的Mac运行alist，但我是小白，只能照着手册说明操作，自己没有相关理论知识。

**Comment by anwen-anyi** [comment]
不是有Mac系统的Alist么

**Reply by Mrered** [reply]
💦💦💦原谅我是纯小白🥹
可以指一下路吗？感谢！🥹

**Comment by ghost** [comment]
[alist-darwin-10.12-amd64.tar.gz](https://github.com/alist-org/alist/releases/download/v3.0.3/alist-darwin-10.12-amd64.tar.gz)
[alist-darwin-10.12-arm64.tar.gz](https://github.com/alist-org/alist/releases/download/v3.0.3/alist-darwin-10.12-arm64.tar.gz)
这两个应该就是macOS的吧

**Reply by anwen-anyi** [reply]
> [alist-darwin-10.12-amd64.tar.gz](https://github.com/alist-org/alist/releases/download/v3.0.3/alist-darwin-10.12-amd64.tar.gz)
> 
> [alist-darwin-10.12-arm64.tar.gz](https://github.com/alist-org/alist/releases/download/v3.0.3/alist-darwin-10.12-arm64.tar.gz)
> 
> 这两个应该就是macOS的吧

是的😊

**Comment by ihxjie** [comment]
可以用docker来装alist，mac对docker的支持还是不错的

**Comment by wickkk119** [comment]
drawin那个文件就是mac版的，你下载解压，跟着官方文档操作就行，跟win操作逻辑一致
